### PR TITLE
templates/rebase: creating initial f${N+1}-coreos-continuous yum repo

### PR DESCRIPTION
**Pull request by dustymabe**
Let's tag in a package into the continuous tag so that the initial
distrepo task will kick off and the yum repo will exist when the first
brave souls try to build COSA against the next major version of Fedora.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/rebase.md [hunk 1]
```diff
@@ -11,6 +11,14 @@ Branching is when a new stream is "branched" off of `rawhide`. This eventually b
 - `f${N+1}-coreos-signing-pending`
 - `f${N+1}-coreos-continuous`
 
+- [ ] Add and tag a package (any package) to the continuous tag. This will create the inital yum repo that's used as input for building the COSA container.
```
[the remark this answers is not in the document]
Comment (dustymabe) [reply .github/ISSUE_TEMPLATE/rebase.md]: fixed.

File: .github/ISSUE_TEMPLATE/rebase.md [hunk 2]
```diff
@@ -11,6 +11,14 @@ Branching is when a new stream is "branched" off of `rawhide`. This eventually b
 - `f${N+1}-coreos-signing-pending`
 - `f${N+1}-coreos-continuous`
 
+- [ ] Add and tag a package (any package) to the continuous tag. This will create the initial yum repo that's used as input for building the COSA container.
+
+- `koji add-pkg --owner ${FAS_USERNAME} f${N+1}-coreos-continuous $PKG`
+    - example: `koji add-pkg --owner dustymabe f36-coreos-continuous fedora-release`
+    - This example uses the [`fedora-release`](https://src.fedoraproject.org/rpms/fedora-release) RPM, but it could be any other.
```
Comment (jlebon) [inline .github/ISSUE_TEMPLATE/rebase.md:18]: Maybe let's link to https://bodhi.fedoraproject.org/updates/?packages=fedora-release instead so hopefully they use an actual build that went through Bodhi and not a random Koji build which might be broken?

And while we're here, might as well make it https://bodhi.fedoraproject.org/updates/?packages=rpm-ostree instead since that's a more likely package we'll tag in at some point.

**Review by jlebon — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/rebase.md (hunk 2 above)
Comment (dustymabe) [reply .github/ISSUE_TEMPLATE/rebase.md:18]: > Maybe let's link to https://bodhi.fedoraproject.org/updates/?packages=fedora-release instead so hopefully they use an actual build that went through Bodhi and not a random Koji build which might be broken?

IIUC the versions shown in the table on https://src.fedoraproject.org/rpms/fedora-release are only ones that filter through Bodhi (not oneoff builds that didn't make it into any release). Am I wrong?

Should I update the link anyway? I kind of like the table view.

> 
> And while we're here, might as well make it https://bodhi.fedoraproject.org/updates/?packages=rpm-ostree instead since that's a more likely package we'll tag in at some point.

What I wanted to do was have them tag a package that actually had the major version in the NVR (i.e. we want to tag a '36' package into a F36 tag). Since this will happen close to branching it's unlikely many packages will have that, but `fedora-release` is one I could think of that definitely should since it will get updated as part of branching.

Comment (jlebon) [reply .github/ISSUE_TEMPLATE/rebase.md:18]: > > Maybe let's link to [bodhi.fedoraproject.org/updates/?packages=fedora-release](https://bodhi.fedoraproject.org/updates/?packages=fedora-release) instead so hopefully they use an actual build that went through Bodhi and not a random Koji build which might be broken?
> 
> IIUC the versions shown in the table on [src.fedoraproject.org/rpms/fedora-release](https://src.fedoraproject.org/rpms/fedora-release) are only ones that filter through Bodhi (not oneoff builds that didn't make it into any release). Am I wrong?

Yup, I think you're right!

> What I wanted to do was have them tag a package that actually had the major version in the NVR (i.e. we want to tag a '36' package into a F36 tag). Since this will happen close to branching it's unlikely many packages will have that, but `fedora-release` is one I could think of that definitely should since it will get updated as part of branching.

This sounds good to me, but maybe we can make the instructions just directly use the corresponding `fedora-release` from that table instead of leaving it up to the user. Or maybe reword as

> Add and tag a package (any package) which is in the stable repos into the continuous tag.

?

Comment (dustymabe) [reply .github/ISSUE_TEMPLATE/rebase.md:18]: > 
> This sounds good to me, but maybe we can make the instructions just directly use the corresponding `fedora-release` from that table instead of leaving it up to the user. Or maybe reword as
> 
> > Add and tag a package (any package) which is in the stable repos into the continuous tag.
> 
> ?

Yeah I just wasn't sure how to automate it.. Though I could probably just find the right tag to query. Either way probably not worth spending much more time on this. Updated the wording with your suggestion ^^.

**Review by jlebon — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/rebase.md (hunk 2 above)
Comment (jlebon) [reply .github/ISSUE_TEMPLATE/rebase.md:18]: SGTM!
